README.md

```diff
@@ -1,20 +1,91 @@
 # Ruby Class Variables And Class Methods Lab
 
-## Outline
+## Objectives
 
+1. Use class variables to keep track of data pertaining to a class. 
+2. Define class methods to expose data pertaining to a class.
+
+## Overview
+
+In this lab, we'll be dealing with a `Song` class. The `Song` class can produce individual songs. Each song has a name, an artist and a genre. We need our `Song` class to be able to keep track of the number of songs that it creates. 
+
+```ruby
 Song.count
+  => 30
+```
+
+We need our `Song` class to be able to show us all of the artists of existing songs:
+
+```ruby
+Song.artists
+  => ["Jay-Z", "Drake", "Beyonce"]
+```
+
+We need our `Song` class to be able to show us all of the genres of existing songs:
+
+```ruby
+Song.genres
+  => ["Rap", "Pop"]
+```
+
+We also need our `Song` class to be able to keep track of the number of songs of each genre it creates. 
+
+In other words, calling:
+
+```ruby
+Song.genre_count
+``` 
+
+Should return something like this;
+
+```ruby
+{"rap" => 5, "rock" => 1, "country" => 3}
+```
+
+Lastly, want our `Song` class to reveal to us the number of songs each artist is responsible for. 
+
+```ruby
+Song.artist_count
+  => {"Beyonce" => 17, "Jay-Z" => 40}
+``` 
+
+We'll accomplish this with the use of class variables and methods. 
+
+## Instructions
+
+* Define your `Song` class such that an individual song is initialized with a name, artist and genre. 
+* There should be an `attr_accessor` for those three attributes. 
+
+```ruby
+99_problems = Song.new("99 Problems", "Jay-Z", "rap")
+
+99_problems.name
+  => "99 Problems"
+
+99_problems.artist
+  => "Jay-Z"
+
+99_problems.genre
+  => "rap"
+```
 
+* Create a class variable, `@@count`. We will use this variable to keep track of the number of new songs that are created from the `Song` class. Set this variable equal to `0`.
+* At what point should we increment our `@@count` of songs? Whenever a new song is created. Your `#initialize` method should use the `@@count` variable and increment the value of that variable by `1`. 
+* Write a class method, `.count`, that returns the total number of songs created. 
```

Comment:
Be consistent on the `#method_name` or `.method_name`. It should be `#method_name`